.github/workflows/bandit.yml

```
@@ -45,8 +45,8 @@ jobs:
           # confidence: # optional, default is UNDEFINED
           # comma-separated list of paths (glob patterns supported) to exclude from scan (note that these are in addition to the excluded paths provided in the config file) (default: .svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg)
           # excluded_paths: # optional, default is DEFAULT
-          # comma-separated list of test IDs to skip
-          # skips: # optional, default is DEFAULT
+          # comma-separated list of test IDs to skip  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)